nntrainer/layers/pooling2d_layer.cpp

```diff
@@ -65,8 +73,8 @@ int Pooling2DLayer::initialize(Manager &manager) {
     }
   }
 
-  unsigned int eff_in_height = in_dim.height() + padding[0] * 2;
-  unsigned int eff_in_width = in_dim.width() + padding[1] * 2;
+  unsigned int eff_in_height = in_dim.height() + pt + pb;
```

Comment:
similar inline functions to conv2d can be helpful here as well.

Reply:
I am thinking to merge some utils in to images (also for preparing thinking extension to conv1d, padding1d).

Let me delay this until merging into v2 :)

Reply:
alright.